pkg/harvester/models/kubevirt.io.virtualmachine.js

```diff
@@ -632,6 +633,21 @@ export default class VirtVm extends HarvesterResource {
     return null;
   }
 
+  get isUnschedulable() {
+    if (this.isBeingStopped) {
+      const condition = this.status?.conditions?.find(c => c.reason === UNSCHEDULABLE);
+
+      if (!!condition) {
+        return {
+          status:  UNSCHEDULABLE,
+          message: condition.message || '',
```

Comment:
Should we need to give the default message if `condition.message` is null to let users know what next step they could take?

Reply:
We can add a default message, but it's not actually used right now.